### DISCUSSION TITLE
Requesting a new Extensibility Event

### DISCUSSION THREAD

**Discussion by matthewhill-ocean** [microsoft/ALAppExtensions#26851] [Q&A] [answered]
Hi,

I've identified a need where a new Extensibility Event would help the development of something I am working on.  I understand I can request for new Events to be added to the Base Application, but I cannot see where I should make this request.

Thanks.

**Comment by NKarolak** [comment, marked as answer]
Right here, under "Issues" :-)
https://github.com/microsoft/ALAppExtensions/issues

**Reply by matthewhill-ocean** [reply]
Got it, thanks!
